workspaces/orchestrator/plugins/orchestrator-backend/src/service/DevModeService.ts

```diff
@@ -155,10 +159,17 @@ export class DevModeService {
       '-v',
       `${resourcesAbsPath}:${SONATA_FLOW_RESOURCES_PATH}:Z`,
     );
+    if (!fs.existsSync(persistenceAbsPath)) {
+      fs.mkdirSync(persistenceAbsPath, { recursive: true });
+    }
+    launcherArgs.push(
+      '-v',
+      `${persistenceAbsPath}:${DEFAULT_SONATAFLOW_PERSISTENCE_PATH}:Z`,
+    );
     launcherArgs.push('-e', 'KOGITO.CODEGEN.PROCESS.FAILONERROR=false');
     launcherArgs.push(
       '-e',
-      `QUARKUS_EMBEDDED_POSTGRESQL_DATA_DIR=${this.connection.persistencePath}`,
+      `QUARKUS_EMBEDDED_POSTGRESQL_DATA_DIR=${DEFAULT_SONATAFLOW_PERSISTENCE_PATH}`,
```

Comment:
need help to understand here: why we don't consider `persistenceAbsPath` anymore?

Reply:
Great idea, will do that change.